ArjunaJTS/jtax/pom.xml

```diff
@@ -335,6 +335,7 @@
                      </property>
                   </systemProperties>
 		  <excludes>
+                    <exclude>**/jts/subordinate/TransactionImpleUnitTest.java</exclude>
```

Comment:
this was added by mistake and needs removing

Reply:
sure, feel free to force push it. we can see the results of the CI while it won't be corrupting the reviews.